resources/public/api/conf/5.0/foreign_property_cumulative_summary_retrieve.yaml

```diff
@@ -0,0 +1,99 @@
+description: |
+  This endpoint enables you to retrieve the cumulative period income and expenses for a foreign property business. A National Insurance number, business ID, and tax year are required. Please note that this endpoint is only available for tax years starting from 2025-26.
+
+  Note: This endpoint only supports submissions for tax year 2025-26 onwards.
+
+  ### Test data
+  Scenario simulation using Gov-Test-Scenario headers is only available in the sandbox environment.
+  
+  | Header Value (Gov-Test-Scenario) | Scenario                                                                 |
+  |----------------------------------|--------------------------------------------------------------------------|
+  | N/A - DEFAULT                    | Simulates success response with Foreign Property.                        |
+  | NOT_FOUND                        | Simulates the scenario where no data is found.                           |
+  | FOREIGN_PROPERTY_FULL_EXPENSES   | Simulates success response of a Foreign property with full expenses.     |
+  | FOREIGN_PROPERTY_CONSOLIDATED    | Simulates the scenario of a Foreign property with consolidated expenses. |
+  | UK_PROPERTY                      | Simulates the scenario where the type of business is incorrect.          |
+  | STATEFUL                         | Performs a stateful retrieve.                                            |
+
+
+tags:
+  - Foreign Property Cumulative Period Summary
+
+parameters:
+  - $ref: './common/pathParameters.yaml#/components/parameters/nino'
+  - $ref: './common/pathParameters.yaml#/components/parameters/businessId'
+  - $ref: './common/pathParameters.yaml#/components/parameters/taxYearFrom2025'
+  - $ref: './common/pathParameters.yaml#/components/parameters/submissionId'
+  - $ref: './common/headers.yaml#/components/parameters/acceptHeader'
+  - $ref: './common/headers.yaml#/components/parameters/authorizationReadScopeHeader'
+  - $ref: './common/headers.yaml#/components/parameters/testScenarioHeader'
+
+security:
+  - User-Restricted:
+      - read:self-assessment
+
+responses:
+  "200":
+    description: Success
+    headers:
+      X-CorrelationId:
+        $ref: './common/headers.yaml#/components/parameters/correlationId'
+      Deprecation:
+        $ref: './common/headers.yaml#/components/parameters/deprecationHeader'
+      Sunset:
+        $ref: './common/headers.yaml#/components/parameters/sunsetHeader'
+      Link:
+        $ref: './common/headers.yaml#/components/parameters/linkHeader'
```

Comment:
The link description appears incomplete. I feel `See the API documentation:` should end with a full stop instead of a colon, like this: `See the API documentation.` See [here](https://github.com/search?q=repo%3Ahmrc%2Fproperty-business-api%20See%20the%20API%20documentation%3A&type=code)